lib/bundler_api/update/job.rb

```diff
@@ -38,6 +39,8 @@ def insert_spec(spec)
     @db.transaction do
       rubygem_insert, rubygem_id = @db_helper.find_or_insert_rubygem(spec)
       version_insert, version_id = @db_helper.find_or_insert_version(spec, rubygem_id, @payload.platform, @payload.checksum, true)
+      info_checksum = @gem_info.info(spec.name)
```

Comment:
This didn't passed the tests because we don't have a test for it yet :-) 
I'll be adding a test and fixing this, it is wrong.